.openpublishing.redirection.json

```diff
@@ -2,19 +2,19 @@
     "redirections": [
         {
             "source_path": "docs/about/index.md", 
-            "redirect_url": "/dotnet/articles/standard/index"
+            "redirect_url": "/dotnet/standard/index"
         },
         {
             "source_path": "docs/about/products.md", 
-            "redirect_url": "/dotnet/articles/standard/components"
+            "redirect_url": "/dotnet/standard/components"
         },         
         {
             "source_path": "docs/cli-preview3/tools/dotnet-nuget-delete.md",
-            "redirect_url": "/dotnet/articles/core/preview3/tools/dotnet-nuget-delete"
+            "redirect_url": "/dotnet/core/preview3/tools/dotnet-nuget-delete"
```

Comment:
Good catch. The redirect existed at one point and got deleted. although the next redirect also exists so it wasn't a big deal.